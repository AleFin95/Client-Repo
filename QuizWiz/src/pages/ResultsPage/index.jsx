import React from 'react';

const ResultsPage = () => {
  return (
    <div className='results'>
      <h1>ResultsPage</h1>
    </div>
  );
};

export default ResultsPage;
