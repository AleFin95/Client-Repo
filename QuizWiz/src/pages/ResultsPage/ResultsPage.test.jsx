import { render, screen } from '@testing-library/react';
import { describe, it,expect } from 'vitest';
import * as matchers from '@testing-library/jest-dom/matchers';

import ResultsPage from './index';

expect.extend(matchers);

describe('ResultsPage Component', () => {
  it('should render an h1 element containing the text "ResultsPage"', () => {
    render(<ResultsPage />);
    const headingElement = screen.getByText('ResultsPage');
    expect(headingElement).toBeInTheDocument();
    expect(headingElement.tagName).toBe('H1');
  });

  it('should match snapshot', () => {
    const { asFragment } = render(<ResultsPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
