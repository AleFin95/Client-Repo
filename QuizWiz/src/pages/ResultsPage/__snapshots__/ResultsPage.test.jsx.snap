// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ResultsPage Component > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="results"
  >
    <h1>
      ResultsPage
    </h1>
  </div>
</DocumentFragment>
`;
